src/auth/hooks/queries/useUser.ts

```diff
@@ -0,0 +1,11 @@
+import { useAuthContext } from "../../context/auth.hook";
+
+export function useUser() {
```

Comment:
I'd prefer `use*` functions (hooks) to have an explicit return type declared - it's much easier for the next person to follow if you do.